src/events/swipe-handler.js

```diff
@@ -26,46 +28,71 @@ export default (
     let translate: number;
     let threshold = 50; //required min distance traveled to be considered swipe
     const restraint = 100; // maximum distance allowed at the same time in perpendicular direction
+    let _slideTo = 0;
+
+    if (!store.getState().isVertical) {
+        $touchTarget.addEventListener('touchstart', (event: TouchEvent) => {
+            const touch = event.changedTouches[0];
+
+            swipeDirection = 'none';
+            threshold = (store.getState().innerSize / options.visibleSlides) / 5;
+            startX = touch.pageX;
+            startY = touch.pageY;
+
+            const translateMatch = $touchTarget.style.transform.match(/-?\d+/i);
+            if (translateMatch && translateMatch[0]) {
+                $touchTarget.classList.remove('animatable');
+                translate = parseInt(translateMatch[0], 10);
+            }
+
+        }, { passive: true });
+
+        $touchTarget.addEventListener('touchmove', (event: TouchEvent) => {
+            const touch = event.changedTouches[0];
+
+            distX = touch.pageX - startX;
+            distY = touch.pageY - startY;
+
+            if (translate !== null && translate !== undefined) {
+                $touchTarget.style.transform = `translateX(${translate + distX}px)`;
+            }
+        }, { passive: true });
+
+        $touchTarget.addEventListener('touchend', (event: TouchEvent) => {
+            if (Math.abs(distX) >= threshold && Math.abs(distY) <= restraint) {
+                swipeDirection = (distX < 0) ? 'left' : 'right';
+            } else if (Math.abs(distY) >= threshold && Math.abs(distX) <= restraint) {
+                swipeDirection = (distY < 0) ? 'up' : 'down';
+                window.scrollTo(0, pageYOffset + 3);
+            } else {
+                $touchTarget.classList.add('animatable');
+                $touchTarget.style.transform = `translateX(${translate}px)`;
+                window.scrollTo(0, pageYOffset + 3);
+            }
+
+
+            const slideWidth = $touchTarget.children[0].style.width.split('px')[0];
```

Comment:
Please use the window.getComputedStyle function like this 

window.getComputedStyle($touchTarget.children[0]).width